### ISSUE TITLE
Update single project detail panel to use new designs drawer on mobile

### ISSUE THREAD

**Issue by TylerMatteo** [NYCPlanning/ae-cp-map#37]
## Acceptance Criteria

- [ ] Take project detail components built in #16 and refactor how they appear on mobile screen sizes to the "[drawer](https://www.figma.com/design/LYHHoPop9l0jpEivk5CFzJ/Capital-Projects-Portal?node-id=1329-23494&t=YvfsIEUhaH1PoS3v-0)" styling from the new designs.

### Blocked by
* #16 - This will use the project detail components built in this Issue
* #14 - We could use this Issue to create a reusable "drawer" component for mobile pages that would be used here.

**Comment by TangoYankee** [comment]
duplicate of #51
